### PR TITLE
lint: increase search depth by one level

### PR DESCRIPTION
**Pull request by davidchambers**
I miss `find`. :)

### CODE REVIEW

**Comment by buzzdecafe** [comment]
:cow2:

File: package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "browserify": "node_modules/browserify/bin/cmd.js test/*.js --outfile lib/test/bundle.js",
     "bookmarklet": "scripts/bookmarklet",
     "clean": "rimraf dist",
-    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js",
-    "jshint": "jshint scripts/bookmarklet scripts/build **/*.js",
+    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js **/**/*.js",
```
Comment (CrossEye) [inline package.json]: Too late to the party once again.  I would rather just:

    "jscs": "jscs scripts/bookmarklet scripts/build .",

which will allow it to work in Windoze too.

File: package.json [hunk 2]
```diff
@@ -36,8 +36,8 @@
...
+    "jshint": "jshint scripts/bookmarklet scripts/build **/*.js **/**/*.js",
```
Comment (CrossEye) [inline package.json]: and here as well.

File: package.json (hunk 1 above)
Comment (davidchambers) [inline package.json]: Wouldn't that match files without the __.js__ extension?

Comment (CrossEye) [inline package.json]: I don't think so.  I mean to test it last night at home.  But I think that means for both jscs and jshint: "Search starting here for any files ending in '.js`."

I did test yesterday on a Windows 7 machine and it worked fine.

**Pull request merged**